frontend_lib/src/component/ShareDownload/ShareDownloadManagement.jsx

```diff
@@ -32,7 +32,7 @@ class ShareDownloadManagement extends React.Component {
     const shareLinkList = props.shareLinkList ? props.shareLinkList : []
     
     return (
-      <div className='shareDownload'>
+      <div>
```

Comment:
use a Fragment instead of a div without any html attributes.
See: https://reactjs.org/docs/fragments.html

you can use the short syntax